svg/attributes/paint-order.json

```diff
@@ -16,7 +16,7 @@
               "version_added": "≤79"
             },
             "firefox": {
-              "version_added": true
+              "version_added": "21"
             },
             "firefox_android": {
               "version_added": true
```

Comment:
```suggestion
              "version_added": "21"
```

Reply:
I don't know about Android version so please leave it unchanged